example.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Nope. I remember now. We moved it up here to test how `urllib` was resolving relative file paths. We should remove it, which may cause tests to fail. If that's the case, then we'll have to monkey with `urllib` some more.